Packs/Twinwave/Integrations/Twinwave/Twinwave.yml

```diff
@@ -0,0 +1,268 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Twinwave
+  version: -1
+configuration:
+- display: Twinwave API token
+  name: api-token
+  required: true
+  type: 4
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- defaultvalue: "10"
+  display: Number of jobs to first fetch
+  name: initial_fetch
+  required: false
+  type: 0
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: all
+  display: Filter incidents by submission source.
+  name: source
+  options:
+  - all
+  - ui
+  - api
+  required: false
+  type: 15
+- display: Filter UI incidents by username. Exact match only. (Cannot use if source
+    is all or api)
+  name: username
+  required: false
+  type: 0
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
```

Comment:
please re order the params:
up top - fetch realted and incident type params
middle - auth params
bottom - proxy and verify